apis/appcat/v1/backup_types.go

```diff
@@ -0,0 +1,146 @@
+package v1
+
+import (
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
```

Comment:
Nitpick: I'd rename these files to something like `vshn_postgres_backup_types.go` or something like that?

Reply:
yeah makes sense